main.go

```diff
@@ -24,6 +29,37 @@ func main() {
 	}
 }
 
+func init() {
+	setupFlagFactory()
+	generateFlags()
+}
+func setupFlagFactory() {
+	// TODO: randomize this, or accept the seed from config.
+	flag.SetFactory(flag.NewFactory(defaultFlagFactorySeed))
+}
+
+func generateFlags() {
```

Comment:
In the flags branch I'm generating the flags based on the session seed, this way

https://github.com/tonyskapunk/yctf/blob/511be24f3ddb3fb2a352809638f70b793d6dc981/models.go#L51-L60

it's important for me that the flag is not always the same, thus having the session as a seed is perfect as it will be always different per-user. Additionaly it helps with #3 and also to what was discussed in #10